sn_faucet/src/main.rs

```diff
@@ -109,6 +109,15 @@ enum SubCmd {
     /// Starts an http server that will send tokens to anyone who requests them.
     /// curl http://localhost:8000/your-hex-encoded-wallet-public-address
     Server,
+    /// Restart the faucet_server from the last breaking point.
+    ///
+    /// Before firing this cmd, ensure:
+    ///   1, The previous faucet_server has been stopped.
+    ///   2, Invalid cash_notes have been removed from the cash_notes folder.
```

Comment:
currently they got removed manually, as so far we can only figure out where is the broken point by manually check the logs.

I think with the later on addin feature of : no move on to next spend till the current one got verified,
then the broken point shall be less to happen, and we may not require to carry out such `restart` much.

Even we need a separate cli to carry out such `prune invalid cash_notes` operations,
it can be done in a separate PR ?